pkg/user/user.go

```diff
@@ -1,32 +1,49 @@
 package user
 
 import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-pg/pg/v9"
+
 	"github.com/chrsep/vor/pkg/auth"
-	"github.com/chrsep/vor/pkg/postgres"
 	"github.com/chrsep/vor/pkg/rest"
-	"github.com/go-chi/chi"
-	"net/http"
+	"github.com/chrsep/vor/pkg/user/store"
 )
 
-type Store interface {
-	GetUser(userId string) (*postgres.User, error)
-	GetSchools(userId string) ([]postgres.School, error)
-}
+var (
+	userStore store.UserStore
+)
```

Comment:
when using dependency injection style, won't the object being called over and over each time we use it? actually this is to prevent the object being called multiple times. but if injection style also do that then no need for this

Reply:
We could just create the object once and pass it on through params. No need to create it multiple times. The DB and store objects are created this way now.

Reply:
hmm i see, oh yes from main.go yea